haystack/nodes/reader/farm.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         proxies=None,
         local_files_only=False,
         force_download=False,
+        use_auth_token: Optional[Union[str,bool]] = None,
```

Comment:
Please add the new param to the docstring below explaining for our end users what kind of token this is and ideally linking some HF documentation on how to obtain it.